package.json

```diff
@@ -9,7 +9,7 @@
     "/dist"
   ],
   "scripts": {
-    "build": "NODE_ENV=production rollup -c",
+    "build": "NODE_ENV=production rollup -c && styleguidist build",
```

Comment:
I remember this was removed in your previous PR, are keeping it?

Reply:
Yea, that was an accident on my end.  Including that last statement just re-builds the Styleguidist static site documentation which gets pushed to the Github repo `/docs` folder.